### DISCUSSION TITLE
feat: 50% less image memory (?)

### DISCUSSION THREAD

**Discussion by MentalGear** [konvajs/konva#1527] [Ideas]
The [Chrome Blog mentioned](https://developer.chrome.com/blog/new-in-chrome-66/#css-paint-api) that loading Imgs in HTML and then in canvas loads the image double in memory.

> Historically, that's meant creating an image tag, then rendering it's contents on to the canvas. Unfortunately that means the browser needs to store multiple copies of the image in memory.

If this is true, it would be good to release the image from HTML/JS once the canvas/Konva has it loaded, by setting the image to null.

It doesn't seem like konva does this yet, so adding a simple img = null after load could yield a 50% better image memory performance.
https://github.com/konvajs/konva/blob/888fbf32549a8ad78584b0011503cdeb0938453a/src/shapes/Image.ts#L43

If you concur I can setup a PR.

**Comment by lavrton** [comment]
It looks like you can do it by yourself. Instead of using an instance of image for `Konva.Image`, you can convert it into bitmap first.
